### DISCUSSION TITLE
Strange orb dataset plot

### DISCUSSION THREAD

**Discussion by kathmcp** [phoebe-project/phoebe2#502] [General]
Hi again, I have added an orb dataset into my code, and it is plotting the attached. I am wondering if it's related to the fact that my observations are from 4 different nights, so is it "jumping" from one place to another when there are gaps in the observations from night to night? 

Or is it related to the way I have read the times into the code from the csv file, which I did with the following code:

lightcurve = pd.read_csv(r"/mnt/c/Users/Asus/AppData/Local/phoebe/BFilterAllFluxTimeFluxErr2.csv")
times_array = np.array(lightcurve['times'])
b_cb.set_value('times', kind='lc', value = times_array)

When adding the orb dataset, I need to enter "times_array" for "compute_times" because if I enter "times" it doesn't work, like so:
b_cb.add_dataset('orb', compute_times= times_array, dataset = 'orb01')
![OrbPlot](https://user-images.githubusercontent.com/81473857/126674017-dda6f465-e869-4474-8653-68d3cb3850df.PNG)

**Comment by kecnry** [comment]
Yep, it's exactly what you thought!  The orbit is exposed and plotted at the times you provided and those points are connected with a straight line by default. If you want a smooth orbit plot, you'll need to provide finer time resolution before calling run_compute (or you can plot just points and no lines).
